src/main/java/seedu/address/logic/parser/tutorial/AddTutorialCommandParser.java

```diff
@@ -1,6 +1,5 @@
 package seedu.address.logic.parser.tutorial;
```

Comment:
Another bug I have just found in class `SampleDataUtil`, method `getSampleReminders()`, the `Datetime` is missing date, only have time.